lib/components/app/responsive-webapp.js

```diff
@@ -212,10 +212,10 @@ class ResponsiveWebapp extends Component {
             <Col className="sidebar" md={4} sm={6}>
               {/*
                 Note: the main tag provides a way for users of screen readers
-                to skip to the primary page content.
+                to skip to the primary page content (tabinde = -1 needed for programmatic navigation skip).
```

Comment:
Typo `tabIndex`

Reply:
Thanks, typos fixed in https://github.com/opentripplanner/otp-react-redux/pull/749/commits/d8aa9b7ec5ba8ff7bae24bdbcd2ebac08c4ec5b2.